src/components/SendMessage.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Image, Pressable, TextInput, View } from "react-native";
+
+import styles from "./styles";
+
+const SEND_ICON = require("../assets/icons/SendIcon.png");
+
+const SendMessage = ({
+  value,
+  handleOnChangeValue,
+  handleOnSendMessage,
```

Comment:
personal taste, but I prefer a naming like this for this functions
```suggestion
  onValueChange,
  onMessageSent,
```